d2bs/kolbot/D2BotMuleLog.dbj

```diff
@@ -420,11 +419,10 @@ MainSwitch:
 		if (FileTools.exists("logs/MuleLog.json")) {
 			obj = JSON.parse(FileTools.readText("logs/MuleLog.json"));
 
-			if (obj.currChar) {
+			if (obj.currChar && obj.currChar !== "Private") { // ControlAction.getCharacters() always returns a non-existent character "Private" at the end of the list
```

Comment:
In my testing all its returning is an empty array. Remove this part and I'll merge this

Reply:
> In my testing all its returning is an empty array. Remove this part and I'll merge this

If you don't have any character, the returned list will indeed be empty.
If you have N characters, then it will contain N+1 characters, the last one will be "Private".

Reply:
No it doesn't. I just again tested this. Private does not exist, getCharacters returns an array. We then use the array to find an remove the current character. Which we are getting from reading the file. No where does this Private object or property value exist.

Reply:
No it doesn't. I just again tested this. Private does not exist, getCharacters returns an array. We then use the array to find an remove the current character. Which we are getting from reading the file. No where does this Private object or property value exist.